aws-ts-webserver/index.ts

```diff
@@ -0,0 +1,37 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+const size = "t2.micro";    // t2.micro is available in the AWS free tier
+
+// Get the id for the latest Amazon Linux AMI
+const ami = aws.ec2.getAmi({
+    filters: [
+        { name: "name", values: ["amzn-ami-hvm-*-x86_64-ebs"] },
+    ],
+    owners: ["137112412989"], // Amazon
+    mostRecent: true,
+}, { async: true }).then(result => result.id);
+
+// create a new security group for port 80
+const group = new aws.ec2.SecurityGroup("web-secgrp", {
+    ingress: [
+        { protocol: "tcp", fromPort: 80, toPort: 80, cidrBlocks: ["0.0.0.0/0"] },
+    ],
+});
+
+// (optional) create a simple web server using the startup script for the instance
+const userData =
+`#!/bin/bash
+echo "Hello, World!" > index.html
+nohup python -m SimpleHTTPServer 80 &`;
+
+const server = new aws.ec2.Instance("web-server-www", {
+    tags: { "Name": "web-server-www" },
+    instanceType: size,
```

Comment:
`pulumi up` run successfully, but you're right, I converted this one a bit too quickly from JS and didn't make it TypeScript native. Will set this to draft and revise.

Reply:
I spot-checked our docs and I couldn't find another example of how I should handle this in a TypeScript way, so I'm leaving it as is (since it works). If there's a more TypeScript way of handling this, just let me know and I'll change it.

Reply:
This is implemented as a relaxed enum, so it'll take a string. For our examples, I think it'd be preferable to show this with an enum: `aws.ec2.InstanceType.T2_Small`